.github/workflows/docbuild.yml

```
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ["3.9", "3.10"]
```

Do you see any reason not to add 3.11? Also, we can probably drop 3.9, based on https://scientific-python.org/specs/spec-0000/

```suggestion
        python-version: ["3.10", "3.11"]
```